src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventProviderTraceListener.cs

```diff
@@ -42,7 +42,7 @@ public string Delimiter
             set
             {
                 if (value == null)
-                    throw new ArgumentNullException("Delimiter");
+                    throw new ArgumentNullException(nameof(Delimiter));
```

Comment:
cc: @iSazonov

Technically `paramName` should be "the name of the parameter that caused the exception" [1], not the name of a property.

As the stack trace will include `set_Delimiter`, it may not be necessary to specify `Delimiter` as the `paramName`.

* [1] https://docs.microsoft.com/dotnet/api/system.argumentnullexception.-ctor?view=net-5.0#System_ArgumentNullException__ctor_System_String_